pset-functions.js

```diff
@@ -147,7 +228,20 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 4:')
 
 // Add your code below this line
-
+function doubleLetters(word){
+  for(i = 0; i <= word.length; i++) {
+        for(var j = i+1; j <= word.length; j++) {
+            if(word[j] == word[i]) {
```

Comment:
I definitely like how you used the nested loop's index to look at the adjacent element in the array; it shows you know how to use the loop's variable to your advantage. Just be a little careful with nested loops, it could really slow down how your code runs. You can use the first loop's index in more than one way; an example of this [here](https://github.com/jsd20200127/hw-02-functions-solution/blob/e87c29073cf7285a5045ec8708561f279474f2d2/pset-functions.js#L225)